app/src/main/java/com/example/fitjournal/journalEntry/domain/WorkoutDetailModel.kt

```diff
@@ -0,0 +1,8 @@
+package com.example.fitjournal.journalEntry.domain
+
+import com.example.fitjournal.core.presentation.model.enums.WorkoutTypeEnum
+
+data class WorkoutDetail(
```

Comment:
@Alex-A-Fit  I am sure this looks familiar to something you just did in one of your PRs. At this point we are just duplicating models. 
if you make yours global then I will get rid of this one. what you think?

Reply:
Im pretty sure i made a model in core already. will need to remove duplicates